TesseractOCR/build_dependencies.sh

```diff
@@ -7,8 +7,9 @@
 
 GLOBAL_OUTDIR="`pwd`/build"
 LOCAL_OUTDIR="./outdir"
-LEPTON_LIB="`pwd`/leptonica-1.71"
 TESSERACT_LIB="`pwd`/tesseract-3.03"
+LEPTON_LIB="`pwd`/leptonica-1.72"
+IMAGE_LIBS="`pwd`/libtiff-ios-master/dependencies"
```

Comment:
It looks like you are assuming the user has this folder in this directory when they call this script. Can you update the instructions for using this build script to reflect any changes in the dependencies (e.g. where they can get that folder)? https://github.com/gali8/Tesseract-OCR-iOS/blob/master/TesseractOCR/README_howto_compile_libaries.md

In fact, since libtiff-ios is a git repo, how do you feel about making it a git submodule in this directory (see https://chrisjean.com/git-submodules-adding-using-removing-and-updating/)? Then the user would just have to run `git submodule init` and `git submodule update` and they would have the folder, and we'd still be able to keep the actual files out of this repo. If you like that idea, be sure to update the README_howto_compile_libaries.md instructions accordingly.

Also it looks like you need to update README_howto_compile_libaries.md to reflect that you are now having the user use leptonica 1.72 instead of 1.71 and update the related download link.